src/js/components/ServiceDetail.js

```diff
@@ -60,6 +59,15 @@ class ServiceDetail extends mixin(InternalStorageMixin, StoreMixin, TabsMixin) {
     });
   }
 
+  componentDidMount() {
+    super.componentDidMount(...arguments);
+    this.checkForVolumes();
+  }
+
+  componentWillUpdate() {
```

Comment:
Nope. No super here please! `componentWillUpdate` is not a method of `React.Component` thus it might throw errors when we're not using `React.DOM` (e.g. in our tests).